src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -104,6 +105,24 @@ interface ILightningService {
 
   defaultPubkey(): Pubkey
 
+  invoiceProbeForRoute({
+    decodedInvoice,
+    maxFee,
+  }: {
+    decodedInvoice: LnInvoice
+    maxFee: Satoshis
+  }): Promise<RawRoute | LightningServiceError>
+
+  noAmountInvoiceProbeForRoute({
+    decodedInvoice,
+    maxFee,
+    amount,
+  }: {
+    decodedInvoice: LnInvoice
+    maxFee: Satoshis
+    amount: Satoshis
+  }): Promise<RawRoute | LightningServiceError>
+
```

Comment:
same issue with naming convention

Reply:
How do you mean? `invoiceProbeForRoute` seems to be a verb that already doesn't exist at graphql layer

Reply:
the methods are returning routes so could be something like `findRouteForInvoice` and `findRouteForNoAmountInvoice`, having invoice and no amount invoice at the beginning does not match the other methods (used naming convention) in the service